app/controllers/admin/articles_controller.rb

```diff
@@ -8,6 +8,7 @@ def create
     
     if @article.save
       redirect_to admin_articles_path
+      flash.notice = 'You have successfully created a new article!'
```

Comment:
This can be passed as an option to `redirect_to` see https://apidock.com/rails/ActionController/Base/redirect_to